src/libfuncs/array.rs

```diff
@@ -59,8 +59,13 @@ where
         ArrayConcreteLibfunc::Append(info) => {
             build_append(context, registry, entry, location, helper, metadata, info)
         }
-        ArrayConcreteLibfunc::PopFront(_) => todo!(),
-        ArrayConcreteLibfunc::PopFrontConsume(_) => todo!(),
+        ArrayConcreteLibfunc::PopFront(info) => {
+            build_pop_front(context, registry, entry, location, helper, metadata, info)
+        }
+        ArrayConcreteLibfunc::PopFrontConsume(info) => {
+            // same signature at sierra level
+            build_pop_front(context, registry, entry, location, helper, metadata, info)
```

Comment:
Would it be better to create the function and forward it from there? In this way, if we ever need to search the libfunc we'll be able to find it by its name.